res/css/views/elements/_Field.scss

```diff
@@ -127,6 +127,7 @@ limitations under the License.
         transform 0.25s ease-out 0s,
         background-color 0.25s ease-out 0s;
     font-size: $font-10px;
+    line-height: normal;
```

Comment:
this is the default value? If something is being overridden on it, please inline comment it

Reply:
I originally added it because of the following reason but I'm not seeing a difference anymore now that we more carefully stop those styles from leaking out via `:not(.mx_AccessibleButton_hasKind)`

> Reset `line-height` so it acts normal regardless of environment (`ContextMenu` has a very large `line-height`)
>
> https://github.com/matrix-org/matrix-react-sdk/pull/7317/files#r765604907

Removed ⏩